docs/Halogen/Component.md

```diff
@@ -273,4 +273,13 @@ install' :: forall s s' f f' g p p'. (Plus g, Ord p) => ParentComponentP s s' f
 A version of [`install`](#install) for use with parent components that
 `peek` on their children.
 
+#### `interpret`
+
+``` purescript
+interpret :: forall s f g g' p. (Functor g') => Natural g g' -> Component s f g p -> Component s f g' p
```

Comment:
Looks good although since `Component` embeds `Free` in the output type (right?) the more general interpreter signature would be `Natural g (Free g')` so you could map one `g` to many `g'`, and have those squashed down to a single level of `Free` inside the Component's query processor.